test/functional/tests/fault_injection/test_cache_insert_error.py

```diff
@@ -21,22 +21,35 @@
 from test_utils.os_utils import Udev
 from test_utils.size import Size, Unit
 
+start_size = Size(512, Unit.Byte)
+stop_size = Size(128, Unit.KibiByte)
+
 
-@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.xfail(
```

Comment:
should pass once OCF is updated:
https://github.com/Open-CAS/open-cas-linux/pull/1341

Reply:
#1341 merged :) Please resolve conflict (and then we can approve review) :)